src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,220 @@
+
+<ImageGrid images={[NestedItemImg, NestedItemStandardsImg]} regularWidth />
+
+## Content-Cover vs. Content-Resize
+
+When a drawer is opened, there are two ways to deal with the main application content.
+
+With content-resize, the main application content resizes its area to accommodate the expanded drawer. This is the default behavior for most drawer variations.
+
+With content-cover, the navigation panel appears on top of the main application content, partially obscuring it. This typically uses a dark overlay behind the drawer to direct the user's focus to the navigation panel and block interaction with the main content. This pattern is most common when using the temporary drawer variant on mobile and tablet applications.
 
 <ImageGrid
-    images={[PersistentImg, PermanentImg]}
-    caption={
-        'When the user hovers on the Navigation Rail, the side navigation will expand temporarily, revealing the text details of each navigation item; when the user moves their cursor away, the whole side navigation panel collapses.'
-    }
+    images={[ContentCoverDesktopImg]}
+    caption={[
+        'Figure 1 displays the normal collapsed state of the menu for desktop.',
+        'Figure 2 displays the open state of the menu with a gray overlay.',
+    ]}
+    regularWidth
+/>
+<ImageGrid
+    images={[ContentCoverTabletImg]}
+    caption={[
+        'Figure 1 displays the normal collapsed state of the menu for tablet.',
+        'Figure 2 displays the open state of the menu with a gray overlay.',
+    ]}
+    regularWidth
 />
 
-<!-- @TODO switch icon-navigation links over to the "Component Recommendations style" instead of a "Interactive Code example" once we have one designed. -->
+## Dividers
 
-<DemoCard
-    repository={'icon-navigation'}
-    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--within-a-drawer-layout'}
-    react={'https://pxblue-components.github.io/react/?path=/story/components-drawer--in-drawer-layout'}
+Divider lines can be used between navigation items in the drawer. They are most effective when used for:
+
+-   separating nested drawer items
+-   separating sections of the drawer contents
+
+<ImageGrid
+    images={[DividerImage]}
+    regularWidth
+    caption={['Avoid using divider lines between every navigation item. These can make the drawer appear busy.']}
 />
 
-The icon navigation bar (a.k.a. "[navigation rail](https://material.io/components/navigation-rail/#anatomy)" on Material.io) is a minimalist approach to the navigation panel. It features a single icon to represent each of the main areas of the application. This navigation panel is typically used on larger screens only, and is expandable to a full side navigation panel on hover or when clicking the menu icon.
+# Design Specifications
+
+<ImageGrid images={[NavigationRailSpecsImg, NavigationRailLabeledImg, PanelSpecsImg, NavigationTabsSpecsImg]} regularWidth />
```

Comment:
![image](https://user-images.githubusercontent.com/8997218/94963036-7ac91900-04c5-11eb-923c-de84796f369b.png)

Drop that "open sans" (bc all our applications are using open sans) and do "14dp 600 / <text color>".  ("600" means "semibold")

Reply:
The app bar spec is already covered by https://pxblue.github.io/patterns/appbar#mobile, so we can remove the "H6" here

Reply:
As a designer 14dp 600 means nothing to me. Are we expecting designers to understand that?

Reply:
Can't we just use the name of the text style (e.g., subtitle2), or is this a one-off?

Reply:
yeah, the same holds for the developers... How about `14dp Semibold / 600`?